### PR TITLE
DTGB-665: Added layouts.

### PR DESCRIPTION
**Pull request by meirege**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have updated the documentation accordingly.
- [ ] I have updated the style guide CHANGELOG accordingly.
- [ ] I have updated the gent_base CHANGELOG accordingly.
- [ ] I have ran gulp axe on the compiled files and found no errors.

### CODE REVIEW

**Review by hstandaert — COMMENTED** [review]
Seems like there's a problem with the image gallery 🤔

screenshot 2019-01-17 at 08 03 06

**Comment by delrueba** [comment]
> Seems like there's a problem with the image gallery thinking
> 
> screenshot 2019-01-17 at 08 03 06

I like it.

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 All Notable changes to `digipolisgent/drupal_theme_gent-base`.
 
+## [Unreleased]
+* DTGB-665: Refactored pages to only show layouts. This shouldn't affect any
```
Comment (hstandaert) [inline CHANGELOG.md]: Not sure if this belongs in the `gent-base` CHANGELOG, since only styleguide was affected by these changes.

**Review by hstandaert — CHANGES_REQUESTED** [review]

File: styleguide/components/41-organisms/header/header.config.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    menu_items: [
+      '<a href=\'#\' class=\'active\'>Home</a>',
+      '<a href=\'#\'>News & events</a>',
+      '<a href=\'#\'>Subpage</a>',
+      '<a href=\'#\'>Another page</a>'
+    ]
+  },
+  variants: [
+    {
+      name: 'logged-in',
```
Comment (delrueba) [inline styleguide/components/41-organisms/header/header.config.js:44]: preview: '@preview'

**Review by delrueba — CHANGES_REQUESTED** [review]

File: styleguide/components/41-organisms/header/header.config.js (hunk 2 above)
Comment (hstandaert) [reply styleguide/components/41-organisms/header/header.config.js:44]:
```suggestion
      name: 'logged-in',
      preview: '@preview',
```

File: CHANGELOG.md (hunk 1 above)
Comment (meirege) [reply CHANGELOG.md]: I removed it. You where right :)

File: styleguide/components/41-organisms/header/header.config.js (hunk 2 above)
Comment (meirege) [reply styleguide/components/41-organisms/header/header.config.js:44]: Added it.

**Review by hstandaert — APPROVED** [review]

**Review by delrueba — APPROVED** [review]

**Pull request merged**